nncf/common/quantization/quantizer_propagation/solver.py

```diff
@@ -443,15 +462,47 @@ def __init__(self,
                         self._operator_allowed_qconfigs_map[op_meta] = default_qconfig_list
         self._active_propagating_quantizers_queue = deque()
         self._finished_propagating_quantizers = []  # type: List[PropagatingQuantizer]
-        self._ignored_scopes = ignored_scopes
-        self._target_scopes = target_scopes
         self._quantizers_waiting_for_branch_merge = QuantizersWaitingForMergeManager()
 
         self._potential_quantizers = {}
         self._num_potential_quantized_activations = 0
         self._quantizable_layer_nodes = quantizable_layer_nodes
         self._post_processing_marker_metatypes = post_processing_marker_metatypes
 
+    @staticmethod
+    def _str_or_list_to_list(list_or_str: Union[List[str], str]) -> List[str]:
+        if list_or_str is None:
+            return []
+        return [list_or_str] if isinstance(list_or_str, str) else list_or_str
+
+    def _filter_by_weight_ignored_target_scopes(self,
+                                                quantizable_layer_nodes: List[QuantizableWeightedLayerNode],
+                                                ignored_scopes_per_group: Dict[QuantizerGroup, List[str]],
+                                                target_scopes_per_group: Dict[QuantizerGroup, List[str]]
+    ) -> Dict[NNCFNodeName, List[QuantizerConfig]]:
+        weight_quantizable_node_names_vs_qconfigs = {}
+        if quantizable_layer_nodes is not None:
```

Comment:
IMHO: This allows you to minimize the number of spaces on the left.
```suggestion
        if quantizable_layer_nodes is None:
            return {}
```